Octokit/Models/Response/PullRequest.cs

```diff
@@ -48,6 +49,11 @@ public PullRequest(Uri url, Uri htmlUrl, Uri diffUrl, Uri patchUrl, Uri issueUrl
         }
 
         /// <summary>
+        /// The universal Id (different from the pull request number) this pull request.
+        /// </summary>
+        public int Id { get; protected set; }
```

Comment:
Got it. You did suggest long. I checked other class like issue, comment, they all using int type id. used int for consistency issue. As you mentioned we have more pullrequest than int can support already, I am going to change it to long again. 

BTW, one of the check Travis CI build failed. I tried to read the log, but did not found any intuitive information. Do you mind to share me some tips on looking at that log?